src/Raven.Server/Utils/CpuUsage.cs

```diff
@@ -96,6 +113,24 @@ private static double CalculateProcessCpuUsage(ProcessInfo currentInfo, ProcessI
             return Math.Min(100, processCpuUsage);
         }
 
+        private static (double MachineCpuUsage, double ProcessCpuUsage) CalculateCpuFromExtensionPoint()
+        {
+            var extensionPoint = new ExtensionPointInfo(CpuUsageExtensionPoint.Data);
+            var machineCpuUsage = extensionPoint.MachineCpuUsage;
+            if (machineCpuUsage < 0)
+            {
+                if (Logger.IsOperationsEnabled)
+                {
+                    Logger.Operations($"Received bad data for cpu usage from extension point : {machineCpuUsage}");
+                }
+
+                machineCpuUsage = 0;
```

Comment:
Not sure that in this case we should use 0, maybe we should fall back to the "normal" calculation

Reply:
@maximburyak But the situation in the first place is the "normal" calculation is wrong.
In my humble opinion, it is better to display 0 that show you something is wrong

Reply:
0 is _not_ wrong.
0 is a valid value

Reply:
So should I return a negative value?

Reply:
Yes, and make sure that this is user visible